tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -344,7 +339,6 @@ def test_it_sets_the_right_javascript_config_settings(
         BasicLTILaunchViews(context, pyramid_request).unconfigured_basic_lti_launch()
 
         assert context.js_config.config == {
-            "mode": "content-item-selection",
```

Comment:
Just spotted the test below re-adds the coverage this line was providing before.